components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -16304,6 +16304,12 @@ public void addAPIRevision(APIRevision apiRevision) throws APIManagementExceptio
                 insertOperationPolicyMappingStatement.executeBatch();
 
                 // Adding to AM_API_CLIENT_CERTIFICATE
+                String getClientCertificatesQuery = SQLConstants.APIRevisionSqlConstants.GET_CLIENT_CERTIFICATES;
+                String driverName = connection.getMetaData().getDriverName();
+                if (driverName.contains("Oracle")) {
+                    getClientCertificatesQuery = SQLConstants.APIRevisionSqlConstants.GET_CLIENT_CERTIFICATES_ORACLE_SQL;
+                }
+
                 PreparedStatement getClientCertificatesStatement = connection
                         .prepareStatement(SQLConstants.APIRevisionSqlConstants.GET_CLIENT_CERTIFICATES);
```

Comment:
Shouldn't `.prepareStatement(SQLConstants.APIRevisionSqlConstants.GET_CLIENT_CERTIFICATES);` be changed to `.prepareStatement(getClientCertificatesQuery);`?